js/main.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-console */
+function getNumber (min, max) {
+  const randomNumber = Math.random();
+
+  if (min > max  ) {
```

Comment:
```suggestion
  if (min > max) {
```
следи за красотой кода)